lib/validators/constants.rb

```diff
@@ -1,5 +1,6 @@
 module Validators
-  EMAIL_FORMAT = /\A[a-z0-9]+([-._][a-z0-9]+)*(\+[^@]+)?@[a-z0-9]+([.-][a-z0-9]+)*\.[a-z]{2,4}\z/i
+  # Support long suffix domain, like: .museum, .fitness
+  EMAIL_FORMAT = /\A[a-z0-9]+([-._][a-z0-9]+)*(\+[^@]+)?@[a-z0-9]+([.-][a-z0-9]+)*\.[a-z]{2,}\z/i
```

Comment:
My idea with this is being more strict about the rules (mainly accepting .com and other common TLDs); you can use `tld: true` for covering cases like this. Convince me that this is a good idea. :)